data/client-side-operations-timeout/deprecated-options.yml

```diff
@@ -0,0 +1,687 @@
+description: "operations ignore maxTimeMS and maxCommitTimeMS options if timeoutMS is set"
```

Comment:
I've altered this test file by removing the spec tests that assert that `socketTimeoutMS` and `wTimeoutMS` are ignored when `timeoutMS` is specified.